### DISCUSSION TITLE
Idea for the freeplay menu and choosing multiple songs.

### DISCUSSION THREAD

**Discussion by Lenancha** [FunkinCrew/Funkin#3642] [Ideas]
I've come up with an idea for the freeplay menu in FNF. Instead of selecting just one song, players could choose to play a set of three or more songs consecutively, or fewer if they prefer, allowing them to enjoy their favorite tunes without being interrupted by the results screen and being sent back to the freeplay menu. This feature could also benefit future characters that have pre-song animations, enhancing the overall experience.

**Comment by JVNpixels** [comment]
Basically a playlist, I see! I saw a couple of mods/engines have this idea, this would be sick!

**Reply by Lenancha** [reply]
That's intriguing. Are you aware of the specific FNF engines/mods that did that? But yes, that's quite accurate to what I'm discussing.

**Reply by JVNpixels** [reply]
Sadly, I do not remember any of the engines or mods, I’ve just seen them on YouTube.

**Reply by Lenancha** [reply]
Aw man. well, thank you for telling me though. And speaking of what else do you think can help this idea btw?

**Reply by JVNpixels** [reply]
Maybe have a little box on the side of the freeplay menu that lists all the songs that you are playing.

**Reply by JVNpixels** [reply]
> Aw man. well, thank you for telling me though. And speaking of what else do you think can help this idea btw?
By the way, you could make this an issue, and select enhancement.

**Reply by Lenancha** [reply]
> Maybe have a little box on the side of the freeplay menu that lists all the songs that you are playing.

That's a good idea tbh.

**Reply by Lenancha** [reply]
> > By the way, you could make this an issue, and select enhancement.

You know what I'll do that. Thank you for talking about this with me lol. Tbh I thought that the ideas was where this should've went since you know it had the word "Idea" on it lol.

**Reply by JVNpixels** [reply]
> > > By the way, you could make this an issue, and select enhancement.
> 
> You know what I'll do that. Thank you for talking about this with me lol. Tbh I thought that the ideas was where this should've went since you know it had the word "Idea" on it lol.

No problem! The developers don't seem to talk much in discussions, they seem to look at issues more since it is more out in the open and is usually the place for requesting new ideas, not new content such as Tutorial (Pico Mix), like you can only request new features.

**Reply by Lenancha** [reply]
I see. And to be fair I don't blame them for using the issues section as you talked about, because you can't just have a game with game breaking bugs, or small yet infuriating glitches getting in the way.
